qrexec/policy/parser.py

```diff
@@ -698,12 +716,25 @@ async def execute(self) -> str:
         if request.source == target:
             raise AccessDenied("loopback qrexec connection not supported")
 
+        if target in {"@adminvm", "dom0", "uuid:00000000-0000-0000-0000-000000000000"}:
+            return f"""\
+user={self.user or 'DEFAULT'}
+result=allow
+target=@adminvm
+autostart={self.autostart}
+requested_target={request.target}"""
+        if target.startswith("@dispvm:"):
+            target_info = request.system_info["domains"][target[8:]]
```

Comment:
This looks a bit inconsistent - you need to look at both `target` and `target_uuid` to get full picture. Wouldn't it be better to have `@dispvm:uuid:...` below in this case?

In any case, please document it in `doc/qrexec-policy-daemon.rst`.

Reply:
I actually implemented your version and then reverted it.  Your version is better, though, so I will un-revert.